driver/examples/configs/baroclinic_c12_from_serialbox.yaml

```diff
@@ -0,0 +1,23 @@
+stencil_config:
```

Comment:
Do we not want to have this in driver/Makefile at all? I thought that we would want to have this in examples because while serialbox may not be a desired option for users, the restart config is expected to be used commonly. So shouldn't we have all the examples?
I think documentation on how to use this yaml file is a good idea (will add this), not just externally but also nice to see easily what options we support and how to use it.

Reply:
What you have right now (the config is here but not the make target) is good. You could also consider putting the config under `driver/tests/` or some other testing directory.